cfme/infrastructure/host.py

```diff
@@ -562,79 +519,90 @@ def get_from_config(provider_config_name):
     credentials = get_credentials_from_config(prov_config['credentials'])
     ipmi_credentials = get_credentials_from_config(prov_config['ipmi_credentials'])
     ipmi_credentials.ipmi = True
-
-    return Host(name=prov_config['name'],
-                hostname=prov_config['hostname'],
-                ip_address=prov_config['ipaddress'],
-                custom_ident=prov_config.get('custom_ident'),
-                host_platform=prov_config.get('host_platform'),
-                ipmi_address=prov_config['ipmi_address'],
-                mac_address=prov_config['mac_address'],
-                interface_type=prov_config.get('interface_type', 'lan'),
-                credentials=credentials,
-                ipmi_credentials=ipmi_credentials)
+    return Host(
+        name=prov_config['name'],
+        hostname=prov_config['hostname'],
+        ip_address=prov_config['ipaddress'],
+        custom_ident=prov_config.get('custom_ident'),
+        host_platform=prov_config.get('host_platform'),
+        ipmi_address=prov_config['ipmi_address'],
+        mac_address=prov_config['mac_address'],
+        interface_type=prov_config.get('interface_type', 'lan'),
+        credentials=credentials,
+        ipmi_credentials=ipmi_credentials
+    )
 
 
 def wait_for_a_host():
-    navigate_to(Host, 'All')
-    logger.info('Waiting for a host to appear...')
-    wait_for(paginator.rec_total, fail_condition=None, message="Wait for any host to appear",
-             num_sec=1000, fail_func=sel.refresh)
+    """Waits for any host to appear in the UI."""
```

Comment:
I believe it should be done in a separate PR. That changes is not related to widgetastic conversion.